examples/v3/createPoolWeighted.ts

```diff
@@ -0,0 +1,78 @@
+import dotenv from 'dotenv';
+dotenv.config();
+
+import { CreatePool, CreatePoolV3WeightedInput } from '@/entities';
```

Comment:
We want to have all the non-test related code imported from the main src so the user can basically copy and paste.